code/game/objects/structures/morgue.dm

```diff
@@ -170,40 +135,103 @@
 	src.attack_hand(CM)
 
 
-/obj/structure/morgue/get_remote_view_fullscreens(mob/user)
+/obj/structure/body_slab/get_remote_view_fullscreens(mob/user)
 	if(user.stat == DEAD || !(user.sight & (SEEOBJS|SEEMOBS)))
 		user.overlay_fullscreen("remote_view", /obj/screen/fullscreen/impaired, 2)
 
+/**
+ * Slabs specifically for the morgue. Show the state of the contents inside of them,
+ * particularly the viability of a body inside.
+ * Can also be marked by pen.
+ */
+/obj/structure/body_slab/morgue
+	name = "morgue"
+	desc = "Used to keep bodies in until someone fetches them."
+
+	var/static/status_descriptors = list(
+		EXTENDED_TRAY = "The tray is currently extended.",
+		EMPTY_MORGUE = "The tray is currently empty.",
+		UNREVIVABLE = "The tray contains an unviable body.",
+		REVIVABLE = "The tray contains a body that is responsive to revival techniques.",
+		NOT_BODY = "The tray contains something that is not a body.",
+		GHOST_CONNECTED = "The tray contains a body that might be responsive."
+	)
+
+/obj/structure/body_slab/morgue/proc/update()
```

Comment:
you cannot define two procs with the same name on the same type path.  you can override it instead by removing the `/proc/` and CI will stop screaming at you.